assistants/cartAssistant.js

```diff
@@ -1,22 +1,58 @@
-import {expect} from "@playwright/test";
-import {PAGES} from "../pages/pages";
-import {Assistants} from "./assistants";
+import { expect } from "@playwright/test";
+import { PAGES } from "../pages/pages";
+import { ASSISTANT, Assistants } from "./assistants";
 
 export class CartAssistant {
+  async popUpAccept(page) {
+    page.on("dialog", async (dialog) => {
+      await dialog.accept();
+    });
+  }
+  async addToCartFirstItem(page) {
+    await page.waitForLoadState("networkidle");
+    await page.locator(PAGES.categoriesPage.itemsTitles).first().click();
+    await page.waitForLoadState("networkidle");
+    const prodName = await page
+      .locator(PAGES.productPage.prodTitle)
+      .textContent();
+    const prodPrice = (
+      await page.locator(PAGES.productPage.prodPrice).textContent()
+    ).split("*")[0];
 
-    async popUpAccept(page,msg){
-        page.on('dialog',async (dialog)=>{
-            expect(dialog.message() === msg);
-            await dialog.accept();
-        })
-    }
-    async addToCartFirstItem(page){
-        await page.waitForLoadState('networkidle');
-        await page.locator(PAGES.categoriesPage.itemsTitles).first().click();
-        await page.waitForLoadState('networkidle');
-        await page.locator(PAGES.productPage.addToCartBtn).click();
-        await Assistants.cartAssistant.popUpAccept(page,'Product added');
+    await page.locator(PAGES.productPage.addToCartBtn).click();
+    await ASSISTANT.cartAssistant.popUpAccept(page);
```

Comment:
```suggestion
    await ASSISTANT.cartAssistant.popUpAccept(page);
```

you can use -> await thist.popUpAccept(page);